### PR TITLE
Test java 17 on rhel8 in CI

### PR DESCRIPTION
**Pull request by liuxueyang**
- Add pljava_test output for Java 17
- Only show feature and interim version of Java in test case
- Run test for Java 17 for rhel8 and ubuntu18.04

### CODE REVIEW

File: pljava-examples/pom.xml [hunk 1]
```diff
@@ -33,8 +33,8 @@
 				<artifactId>maven-compiler-plugin</artifactId>
 				<version>2.5.1</version>
 				<configuration>
-					<source>1.6</source>
-					<target>1.6</target>
+					<source>1.7</source>
```
Comment (beeender) [inline pljava-examples/pom.xml:36]: why do we need to change this?

Comment (liuxueyang) [reply pljava-examples/pom.xml:36]: Good point. More investigation is needed here, thank you.

**Review by beeender — APPROVED** [review]

File: concourse/scripts/test_pljava.sh
[the remark this answers is not in the document]
Comment (beeender) [reply concourse/scripts/test_pljava.sh:1]: not in this PR. If it is needed, do it in another PR. Let's try to make PRs smaller.

[the remark this answers is not in the document]
Comment (liuxueyang) [reply concourse/scripts/test_pljava.sh:1]: I'll do this in another PR.

**Pull request merged**